winui/Masked-TextBox/Getting-Started.md

```diff
@@ -90,4 +90,49 @@ maskedTextBox.Mask = "[A-Za-z0-9._%-]+@[A-Za-z0-9]+\\.[A-Za-z]{2,3}";
 {% endhighlight %}
 {% endtabs %}
 
-![RegEx mask in WinUI MaskedTextBox](MaskedTextBox_Images/winui_regex_mask.png)
\ No newline at end of file
+![RegEx mask in WinUI MaskedTextBox](MaskedTextBox_Images/winui_regex_mask.png)
+
+## Setting the prompt character
+
+Displays prompt character for the absence of your input in the mask and its default value is ‘_’. You can set the custom prompt character using `PromptChar` property.
```

Comment:
@Vignesh-SF3580 Every single mask displays a prompt character ('_') in the absence of your input. You can customize this prompt character by using the `PromptChar` property."

Reply:
Changed the content for setting the prompt character.